spec/apply_for_a_course_spec.rb

```diff
@@ -0,0 +1,15 @@
+RSpec.describe "Apply for a course", type: :feature do
+  scenario "User can apply for a course" do
+    visit "https://qa.find-postgraduate-teacher-training.service.gov.uk/course/2Z3/B275"
```

Comment:
Should the find URL also be in `.env`? I realise I forgot to move it in my other PR.